definitions/npm/express_v4.x.x/flow_>=v0.25.x/express_v4.x.x.js

```diff
@@ -73,7 +73,7 @@ declare module 'express' {
     location(path: string): this;
     redirect(url: string, ...args: Array<void>): this;
     redirect(status: number, url: string, ...args: Array<void>): this;
-    render(view: string, locals?: mixed, callback: (err?: ?Error) => mixed): this;
+    render(view: string, locals?: mixed, callback: (err?: ?Error, html: string) => mixed): this;
```

Comment:
Good catch. Looks like `html` is omitted if there's an error: https://github.com/expressjs/express/blob/master/lib/application.js#L581

And it's always called with at least the first arg. So I'll change the signature.

The Response render just calls to app render so I'll share the type too.